packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -735,23 +780,62 @@ export class IntervalCollectionView<TInterval extends ISerializableInterval> ext
         return interval;
     }
 
-    public deleteInterval(serializedInterval: ISerializedInterval, local: boolean, op: ISequencedDocumentMessage) {
-        const interval = this.localCollection.removeInterval(serializedInterval.start, serializedInterval.end);
+    public delete(start: number, end: number) {
+        for (let interval: TInterval | undefined = this.localCollection.findInterval(start, end);
+                interval;
+                interval = this.localCollection.findInterval(start, end)) {
+            this.deleteExistingInterval(interval, true, undefined);
+        }
+    }
+
+    public deleteInterval(
+        serializedInterval: ISerializedInterval, local: boolean, op: ISequencedDocumentMessage) {
+        const id = serializedInterval.properties?.[reservedIntervalIdKey];
+        if (id === undefined) {
+            // Delete all (start, end).
+            const start = serializedInterval.start;
```

Comment:
given above, which will send an delete op per interval, this seems wrong as well

Reply:
it looks like we will send and op per interval at a range, and then delete all intervals for each op. as i mention above, i dislike this type of behavior, as it is hard to get eventually consistent. i would make operations granular, specifcally 1 op per interval

Reply:
Yeah, the ops over the wire are granular. In the delete path, only deleteExistingInterval sends out a serialized op.